english/eating-walls/find-a-toilet/door/bear/too-many-marshmallows/get-eaten.md

```diff
@@ -0,0 +1 @@
+The bear turns round, growls at you, and then in a fit of rage grasps you in a bear hug and starts eating you from the head down.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.